src/scales/scale.logarithmic.js

```diff
@@ -48,11 +48,14 @@ module.exports = function(Chart) {
 						if (valuesPerType[meta.type] === undefined) {
 							valuesPerType[meta.type] = [];
 						}
-
+                        
 						helpers.each(dataset.data, function(rawValue, index) {
 							var values = valuesPerType[meta.type];
 							var value = +me.getRightValue(rawValue);
-							if (isNaN(value) || meta.data[index].hidden) {
+                            if(value === 0){
+                                dataset.data[index] = null;  // don't show y-value
```

Comment:
I'm not sure that the scale should be changing the real data value. Does this have to be changed or can value simply be set to `null`.